src/include/parser/expression_defs.h

```diff
@@ -38,6 +40,8 @@ enum class ExpressionType : uint8_t {
   CONJUNCTION_AND,
   CONJUNCTION_OR,
 
+  COLUMN_VALUE,
```

Comment:
How is this different from existing `ExpressionType::VALUE_TUPLE`?

Reply:
Value_tuple is the expression type of derivedValueExpression, and column_value for ColumnValueExpression